src/objects/domain_obj.f90

```diff
@@ -2158,6 +2159,9 @@ subroutine read_domain_shape(this, options)
         call this%u_grid%set_grid_dimensions(       nx_global, ny_global, nz_global, nx_extra = 1)
         call this%v_grid%set_grid_dimensions(       nx_global, ny_global, nz_global, ny_extra = 1)
 
+        call this%grid_smooth_nsquared%set_grid_dimensions(nx_global, ny_global, nz_global, &
+             halo_width = (nsmooth+2)*2)
```

Comment:
which nsmooth is this? there is a separate smoothing window for nsq than there is for the wind field, we need to make sure we can keep these separate.

Reply:
Looks like it gets set ~20 lines above, right [here](https://github.com/scrasmussen/icar/blob/a4283684d7609e28e003387eb4a33293e9361637/src/objects/domain_obj.f90#LL2144C12-L2144C12):
```
        nsmooth = max(1, int(options%parameters%smooth_wind_distance / options%parameters%dx))
```
I could move that statement down so it's next the `grid_smooth_nsquared`, nothing above it uses it and the grouping of code would be a bit clearer

Reply:
I see what you mean, that is the wrong `nsmooth`, I'll find where the other one is set

Reply:
Ok, found [it](https://github.com/scrasmussen/icar/blob/a4283684d7609e28e003387eb4a33293e9361637/src/physics/linear_winds.f90#L1145).  Will update the PR
```
stability_window_size = options%lt_options%stability_window_size        ! window to average nsq over
```